### ISSUE TITLE
2023-09-07 Patch Release Plan

### ISSUE THREAD

**Issue by trackleft** [az-digital/arizona-bootstrap#920]
### Bugfixes
- #899
- #901

### Build tool updates
- #883
- #884
- #882
- #881
- #880
- #867
- #866
- #873
- #852
- #829
- #792

**Comment by trackleft** [comment]
Release notes snippet
```
 git log v2.0.24..HEAD --pretty='format:- %s'
 ```

```
- az-digital/az_quickstart#2549 fix mobile horizontal scroll. (#901)
- Closes #887: Address off-canvas menu usability issues on iOS (#899)
- Bump @babel/core from 7.22.9 to 7.22.10 (#883)
- Bump eslint from 8.46.0 to 8.47.0 (#884)
- Bump @babel/preset-env from 7.22.9 to 7.22.10 (#882)
- Bump autoprefixer from 10.4.14 to 10.4.15 (#881)
- Bump @babel/cli from 7.22.9 to 7.22.10 (#880)
- Bump rollup from 3.27.2 to 3.28.0 (#867)
- Bump hugo-bin from 0.112.1 to 0.113.0 (#866)
- Update 2.x branch (#873)
- Closes #849: backport late July dev dependency version bumps. (#852)
- Closes #809: Backport dev dependency July version number bumps. (#829)
- Closes #790: backport dev dependency updates to the 2.x branch. (#792)
```

**Comment by trackleft** [comment]
Version 2.0.25 released https://github.com/az-digital/arizona-bootstrap/releases/tag/v2.0.25
